stats.nim

```diff
@@ -55,10 +55,10 @@ proc countRead*(rs: var bamStats, rec: Record, chrom: string = "") =
   rs.readLengthHist.inc(rec.b.core.l_qseq)
 
   # This field is dubious as it depends on the order of traversing the bam file
-  rs.lastReadPos = rec.start
+  rs.lastReadPos = rec.start.int
```

Comment:
These are the only changes I needed to make to get it to compile, there weren't other merge conflicts